udata/tests/api/test_organizations_api.py

```diff
@@ -637,6 +637,35 @@ def test_suggest_organizations_homonyms(self, api):
         for suggestion in response.json:
             assert suggestion['name'] == 'homonym'
 
+    def test_suggest_organizations_acronym(self, api):
+        '''Should suggest organizations based on acronym'''
+
+        for i in range(3):
+            OrganizationFactory(
+                name=faker.word(),
+                acronym=f'TEST{i}' if i % 2 else faker.word(),
```

Comment:
`TEST` is a risky one since faker.word() sometimes output words that contains `test`.
Maybe replace by a rarer word: `udata`, `opendata`, etc.?